docs/ad/index.md

```diff
@@ -40,36 +45,54 @@ A feature is the field in your index that you want to find anomalies for. A dete
 You can add a maximum of five features for a detector.
 {: .note }
 
-1. Select the **Enable feature** check box.
-2. Add the **Feature name**.
-3. For **Feature type**, choose **Field aggregation**, and then choose the aggregation method. Or choose **Custom expression**, and add in your own JSON aggregation query.
-4. Choose a **Field**.
-5. Choose **Save**.
+1. On the **Features** page, select **Add features**.
+1. Enter the **Name** of the feature.
+1. For **Find anomalies based on**, choose the method to find anomalies. For **Field Value** menu, choose the **field** and the **aggregation method**. Or choose **Custom expression**, and add in your own JSON aggregation query.
+1. Preview sample anomalies and adjust the feature settings if needed.
+- For sample previews, the anomaly detection plugin selects a small number of data samples, for example, one data point for every 30 minutes, and uses interpolation to estimate the remaining data points to approximates the actual feature data. It loads this sample dataset into the detector. The detector uses this sample dataset to generate a sample preview of anomaly results.
+Examine the sample preview and use it to fine-tune your feature configurations, for example, enable or disable features, to get more accurate results.
+1. Choose **Save**.
+1. Choose between automatically starting the detector (recommended) or manually starting the detector at a later time.
 
 ### Step 3: Observe the results
 
-The **Total anomalies** graph plots the anomaly grade with the corresponding measure of confidence.
+Choose the **Anomaly results** tab.
+
+You will have to wait for some time to see the anomaly results.
+The detector needs at least 6 data points in 8 continuous intervals (shingles) to initialize.
+So, if the detector interval is 10 min, the detector might take more than an hour to initialize.
+If you see the detector pending in initialization for a long time, increase the detector interval to fasten the process.
+Or, use the [profile detector](./api#profile-detector) operation to make sure you have sufficient data in a shingle size of 8.
+
+![Anomaly detection results](../images/ad.png)
```

Comment:
How about we change to this one which shows both anomaly result and alert history?

![Screen Shot 2020-05-12 at 4 39 04 PM](https://user-images.githubusercontent.com/49084640/81756119-ae9c9700-946f-11ea-9030-dcd037f7700d.png)